docs/source/api/apollo-server.mdx

```diff
@@ -228,6 +229,26 @@ Available in Apollo Server v3.4.0 and later.
 </td>
 </tr>
 
+<tr>
+<td>
+
+##### `cache`
+
+`KeyValueCache | "bounded"`
+</td>
+<td>
+
+A `KeyValueCache` which Apollo Server uses for a number of features like APQs and full response caching. This cache is also provided to DataSources and plugins.
+
+By default, the cache is unbounded. We don't recommend this, since a malicious client can run your server out of memory and cause it to crash by filling it with APQs.
+
+If you don't want to configure your own cache, you should set `cache: "bounded"`. The bounded cache is an [`InMemoryLRUCache`](https://www.npmjs.com/package/@apollo/utils.keyvaluecache) with a default size of roughly 30MiB.
```

Comment:
Removing the first sentence because we now address this above:
```suggestion
The default bounded cache is an [`InMemoryLRUCache`](https://www.npmjs.com/package/@apollo/utils.keyvaluecache) with a default size of roughly 30MiB.
```

Reply:
I agree that the duplication doesn't help but I think we should merge it with the previous paragraph and write something like "This bounded cache".  Otherwise it's too easy to misread this as describing the default if you specify nothing. It's not a default, it's what `"bounded"` does.